src/Symfony/Component/Messenger/Exception/HandlerFailedException.php

```diff
@@ -20,7 +20,7 @@ class HandlerFailedException extends RuntimeException implements WrappedExceptio
     private Envelope $envelope;
 
     /**
-     * @param \Throwable[] $exceptions
+     * @param array<string, \Throwable> $exceptions The name of the handler should be given as key
```

Comment:
`WrappedExceptionsInterface` needs a similar update, I guess, and `DelayedMessageHandlingException` then needs to also set the handler as key.

Reply:
Oh, and if it's a "should", then it means the type should be `array<\Throwable>`

Reply:
Should I apply @chalasr's [solution](https://github.com/symfony/symfony/pull/51331#issuecomment-1743650831) to `DelayedMessageHandlingException` then?

Reply:
WDYM? getWrappedExceptions is new to 6.4

Reply:
There isn't an easy way to get the handler name to use as the key for `DelayedMessageHandlingException`. The handlers aren't available when creating this exception.

Reply:
ok, thanks for checking

the type should then definitely be changed:
```suggestion
     * @param array<\Throwable> $exceptions The name of the handler should be given as key
```